File: create-wrong-icon.js
// Create wrong_mark.png - Red X icon for wrong cells
const fs = require('fs');
const { createCanvas } = require('canvas');

const size = 128;
const canvas = createCanvas(size, size);
const ctx = canvas.getContext('2d');

// Transparent background
ctx.clearRect(0, 0, size, size);

// Draw red X
ctx.strokeStyle = '#FF0000';
ctx.lineWidth = 20;
ctx.lineCap = 'round';

const padding = 20;
// Top-left to bottom-right
ctx.beginPath();
ctx.moveTo(padding, padding);
ctx.lineTo(size - padding, size - padding);
ctx.stroke();

// Top-right to bottom-left
ctx.beginPath();
ctx.moveTo(size - padding, padding);
ctx.lineTo(padding, size - padding);
ctx.stroke();

// Save
const buffer = canvas.toBuffer('image/png');
fs.writeFileSync('modules/paint-canvas/android/src/main/res/drawable/wrong_mark.png', buffer);
console.log('✅ wrong_mark.png created!');

File: create-wrong-icon.py
from PIL import Image, ImageDraw

# Create 128x128 transparent image
size = 128
img = Image.new('RGBA', (size, size), (0, 0, 0, 0))
draw = ImageDraw.Draw(img)

# Draw red X
padding = 20
line_width = 20

# Top-left to bottom-right
draw.line([(padding, padding), (size-padding, size-padding)], fill='#FF0000', width=line_width)

# Top-right to bottom-left
draw.line([(size-padding, padding), (padding, size-padding)], fill='#FF0000', width=line_width)

# Save
img.save('modules/paint-canvas/android/src/main/res/drawable/wrong_mark.png')
print('✅ wrong_mark.png created!')
